scripts/jest/matchers/reactTestMatchers.js

```diff
@@ -35,6 +35,7 @@ function toMatchRenderedOutput(ReactNoop, expectedJSX) {
   if (typeof ReactNoop.getChildrenAsJSX === 'function') {
     const Scheduler = ReactNoop._Scheduler;
     assertYieldsWereCleared(Scheduler, toMatchRenderedOutput);
+    assertConsoleLogsCleared();
```

Comment:
nit: Should probably move this into `assertYieldsWereCleared` for parity with `assertYieldsWereCleared` in `ReactInternalTestUtils`.